README.md

```diff
@@ -114,6 +114,48 @@ To run `DefaultAzureCredentailAdapterTest`, ensure you have `.env` file created
 
 Once you have the `.env` file configured, run the test using JUnit 5 runner.
 
+## TypeScript
+
+### DefaultAzureCredentialAdapter.ts
+
+The `DefaultAzureCredentialAdapter` class provides a simple bridge to use `DefaultAzureCredential` from `@azure/` package name in `azure-` SDKs. 
+
+To use this type, just copy `DefaultAzureCredentialAdapter.ts`, `package.json`, and `tsconfig.json` file located in `js` directory into your application and install packages in `package.json`.
+
+After you have created this type, you can reference it in your code as shown below:
+
+```TypeScript
+# Example for azure-mgmt-keyvault client
+const cred = new DefaultAzureCredentialAdapter();
+const client = new KeyVaultManagementClient(cred, subscriptionId);
+```
+
+The above code will provide an instance of `KeyVaultManagementClient` from which you can access the Key Vault service..
+
+#### Testing DefaultAzureCredentialAdapter
+
+This repository has a test that gets an existing Key Vault in a given resource group.
+
+To run this test, ensure you have `.env` file created and accessible from the root of your repo. Your `.env` file should have the following properties set:
+
+- AZURE_SUBSCRIPTION_ID
+- AZURE_RESOURCE_GROUP_NAME
```

Comment:
Update these names to match what is in .env.tmp.  See my other comment for correct names.